web/js/tablesorter.js

```diff
@@ -25,12 +25,15 @@ $(document).ready(function() {
             // extra css class name (string or array) added to the filter element (input or select)
             filter_cssFilter: [
                 'form-control',
-                'form-control',
+                'form-control custom-select',
                 'form-control custom-select', // select needs custom class names :(
```

Comment:
Was bedeutet das?

Reply:
tja, da habe ich etwas "getrickst".
Das Tablesort möchte ich gerne für alle großen Tabelle benutzen. Dabei nehme ich an, dass in der ersten Spalte immer die Nummer steht und damit danach in jeder beliebigen Spalte das Select funktioniert, steht ab der 2. Spalte immer "custom-select". Sonst würde das auswählen eines Stamms aus einer Liste nicht funktionieren.
"Getrickst" habe ich damit, weil nun überall (ab der 2. Spalte) der Auswahlpfeil zu sehen ist, obwohl in den meisten Fällen keine Stämme o.ä. zur Auswahl stehen. Man hat also ein input Feld mit Auswahlpfeilen. Ich hoffe, dass du damit leben kannst, denn sonst wird es aufwändiger den Tablesorter einzubinden.

Reply:
Man müsste dann füh jede Tabelle eine Funktion erstellen oder?

Reply:
so in etwa stell ich mir das vor. D.h. eine "$(document).ready(function() {" für jeden Tablesorter, in je einer eigenen JS Datei, um es ordentlich zu machen.
Meiner Ansicht nach leider mehr die Übersicht, als dass der Nutzen steigt. Und es funktioniert so.

Reply:
Ja das stimmt. Das können wir ändern, wenn es wichtig wird! Wollen wir uns dafür vielleicht ein IceBox Issue erstellen. Damit wir das nicht ganz vergessen?

Reply:
Wie du magst. Ich sehe nicht die Notwendigkeit dafür.